libevm/VMOpt.cpp

```diff
@@ -63,14 +63,16 @@ void VM::copyCode(int _extraBytes)
 
 void VM::optimize()
 {
+	copyCode(33);
```

Comment:
Probably we could resize m_codeSpace after the loop if it's really needed... Yeah, that was what you meant I think, ok I'll add it.